sympy/matrices/tests/test_normalforms.py

```diff
@@ -0,0 +1,19 @@
+from sympy import Symbol, Poly
+
+from sympy.matrices import Matrix
+from sympy.matrices.normalforms import *
```

Comment:
Let's try to follow Python's Zen: `Explicit is better than implicit`
and import `smith_normal_form` and `smith_normal_invariants` explicitly here.